packages/coordinator/package.json

```diff
@@ -12,15 +16,14 @@
   "scripts": {
     "hardhat": "hardhat node",
     "build": "nest build",
-    "start": "nest start",
-    "start:dev": "nest start --watch",
-    "start:prod": "node dist/main",
+    "start": "node --import 'data:text/javascript,import { register } from \"node:module\"; import { pathToFileURL } from \"node:url\"; register(\"ts-node/esm\", pathToFileURL(\"./\"));' ./ts/main.ts",
```

Comment:
I think we need to find a way how to run commands without specifying these imports for each command.